src/Broadcasts/View.elm

```diff
@@ -140,7 +140,7 @@ broadcastContent time broadcast =
     let
         fractionTimeLeft : Float
         fractionTimeLeft =
-            ((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour)
+            1 + (((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour))
```

Comment:
do some debugging/logging. elm math is terrible and gives a negative number on `(Date.toTime broadcast.createdAt) - time`, which is actually the negative complement of the fraction of time left